source/apiVolontaria/apiVolontaria/tests/tests_view_ResetPassword.py

```diff
@@ -240,18 +243,19 @@ def test_create_new_token_without_email_service(self):
         """
         data = {
             'username': self.user.username,
+            'email': self.user.email
```

Comment:
idem

Reply:
done

Reply:
Tu transmet toujours "email" plutôt que "username" ici

Reply:
Idem, marked as resolved but not resolved.